extensions/guacamole-auth-http/.gitignore

```diff
@@ -0,0 +1,9 @@
+target/
+pom.xml.tag
```

Comment:
So ... what are all these files being ignored? A typical `.gitignore` just contains:

    target/
    *~

Though even that is probably not necessary due to the top-level `.gitignore` of guacamole-client.